filter_linux.go

```diff
@@ -172,6 +172,14 @@ func (h *Handle) FilterAdd(filter Filter) error {
 		if filter.ClassId != 0 {
 			nl.NewRtAttrChild(options, nl.TCA_U32_CLASSID, nl.Uint32Attr(filter.ClassId))
 		}
+		if filter.Divisor != 0 {
+			// Validate if the divisor is a power of two.
+			if (filter.Divisor-1)&filter.Divisor != 0 {
+				return fmt.Errorf("Illegal divisor, which must be a power of two.")
```

Comment:
Please follow the convention of error string starting with lower case
`"illegal divisor: %d. Must be a power of 2.", filter.Divisor`

Reply:
Fixed.